scripts/.env.instantiate.example

```diff
@@ -0,0 +1,26 @@
+NEUTRON_RPC="https://rpc-falcron.pion-1.ntrn.tech:443"
+GAS_PRICES=0.02
+NEUTRON_CHAIN_ID="pion-1"
+TARGET_CHAIN_ID="test-2"
+KEYRING_BACKEND="test"
+DEPLOY_WALLET="drop-deploy"
+MIN_NTRN_REQUIRED=10
+TARGET_SDK_VERSION="0.47.10"
+TARGET_BASE_DENOM="uatom"
+NEUTRON_SIDE_TRANSFER_CHANNEL_ID="channel-788"
+IBC_REGISTER_FEE="1000000"
+
+INITIAL_VALIDATORS='[{"valoper_address":"cosmosvaloper1...","weight":10},{"valoper_address":"cosmosvaloper1...","weight":10}]'
+
+factory_code_id=
+core_code_id=
+token_code_id=
+distribution_code_id=
+staker_code_id=
+validators_set_code_id=
+puppeteer_code_id=
+rewards_manager_code_id=
+strategy_code_id=
+withdrawal_manager_code_id=
+withdrawal_voucher_code_id=
+pump_code_id=
```

Comment:
Do I really have to manually fill these variables after I run `./upload.sh`? That's not good because we will eventually need to run these scripts from tests.

Reply:
Yes, it was made like this to have consistency between calls in different scripts and to make it possible to build scripts for different actions while keeping internal data consistent.
Another reason of having this variables is to upload contracts code once and instantiate protocol many time with same binary.
So you can upload contracts using `upload_contracts.bash` then write down uploaded binaries codes and run instantiate many times with different initial settings.
if you want to run these scripts from tests you can set env varaibles from tests, np. Also `deploy.bash` is using same variable naming schema.